x/stakeibc/keeper/icacallbacks_delegate.go

```diff
@@ -51,6 +51,7 @@ func DelegateCallback(k Keeper, ctx sdk.Context, packet channeltypes.Packet, txM
 	}
 	k.Logger(ctx).Info(fmt.Sprintf("DelegateCallback %v", delegateCallback))
 	hostZone := delegateCallback.GetHostZoneId()
+	k.Logger(ctx).Info(fmt.Sprintf("[DELEGATION] success on %s", hostZone))
```

Comment:
nit: move to the end of the callback function

Reply:
done